packages/datatrak-web/src/features/Survey/Components/SurveyQuestion.tsx

```diff
@@ -86,10 +86,10 @@ export const SurveyQuestion = ({
   }
 
   // If the question dictates the visibility of any other questions, we need to update the formData when the value changes, so the visibility of other questions can be updated in real time. This doesn't happen that often, so it shouldn't have too much of a performance impact, and we are only updating the formData for the question that is changing, not the entire formData object.
-  const handleOnChange = e => {
+  const handleOnChange = (newValue: unknown) => {
     if (updateFormDataOnChange) {
       setFormData({
-        [name]: e?.target ? e.target.value : e?.value,
```

Comment:
So there is a slightly weird case here, I think it's possibly in the autocomplete question and maybe also the entity question, where the onChange event returns an object with `value` as opposed to a typical change event object, so we would need to handle that case as well

Reply:
@alexd-bes thanks for noting this, my original approach wasn't working with autocomplete questions as you predicted.

I decided to go with a slightly different approach this time. Now `onChange` takes two arguments, the first is the input value and the second is the raw value. That way we can let `react-form-hook` still do it's thing while also passing in the raw value to `updateFormData()`